edge/pkg/edged/edged.go

```diff
@@ -273,13 +273,13 @@ func (e *edged) handlePod(op string, content []byte, podCfg *config.PodConfig) (
 		switch op {
 		case model.InsertOperation:
 			adds := &kubelettypes.PodUpdate{Op: kubelettypes.ADD, Pods: pods, Source: kubelettypes.ApiserverSource}
-			podCfg.EdgedCh <- *adds
+			podCfg.Channel(kubelettypes.ApiserverSource) <- *adds
```

Comment:
we can get the  `podCfg.Channel(kubelettypes.ApiserverSource)` once and set it to the member of edged to reduce the call for `podCfg.Channel(kubelettypes.ApiserverSource)`

Reply:
done